model/textparse/protobufparse.go

```diff
@@ -287,7 +292,7 @@ func (p *ProtobufParser) Type() ([]byte, model.MetricType) {
 // Must only be called after Next returned a unit entry.
 // The returned byte slices become invalid after the next call to Next.
 func (p *ProtobufParser) Unit() ([]byte, []byte) {
-	return p.metricBytes.Bytes(), []byte(p.mf.GetUnit())
+	return p.seriesBytes.Bytes(), []byte(p.dec.GetUnit())
```

Comment:
Does seriesBytes fulfill the expectation of the caller?
I think it's expecting a metric name.

Reply:
It's a metric name in this buffer for units. Maybe I should rename to `buf` then or so (: